bin/travis.sh

```diff
@@ -4,7 +4,7 @@ set -e
 npm test
 ./bin/coveralls.sh
 if [ ${RUN_LINTERS:="no"} == "yes" ]; then
-    ./node_modules/.bin/eslint src/**/*.js;
+    npm run lint
```

Comment:
The 'lint' script doesn't set the max-warnings flag which I think is useful and important on our CI runs.

Reply:
Damn.... the diff changed after I submitted my comment. I guess the branch was rebased? At some point in my branch I had two lines running eslint and prettier independently with slightly different arguments to what's set in the 'lint' script field within the package manifest.